packages/bitcoin/src/utils/bitcoin.utils.ts

```diff
@@ -359,7 +359,3 @@ export function getBitcoinInputValue(input: TransactionInput) {
   // logger.warn('Unable to find either `witnessUtxo` or `nonWitnessUtxo` in input. Defaulting to 0');
   return 0;
 }
-
```

Comment:
@alexp3y - refactoring this away as its no longer used but I can keep it if you think it could be useful